### PR TITLE
Script tag cleanup

### LINKED ISSUE #951: Normalize <script> tag syntax
https://github.com/department-of-veterans-affairs/vets-website/blob/master/_includes/header.html#L159

This particular script tag uses by the `type=text/javascript` attribute as well as the

```
//<~[CDATA[ 
//]]>
```

The other script tags don't.  We should normalize.  Per

http://stackoverflow.com/questions/2626563/leaving-out-type-text-javascript-language-javascript

We are using the html5 doctype definition in our header: https://github.com/department-of-veterans-affairs/vets-website/blob/master/_includes/header.html#L1

making the type attribute unnecessary for text/js.  

Per http://stackoverflow.com/questions/66837/when-is-a-cdata-section-necessary-within-a-script-tag the CDATA declaration is defunct along with XHTML.

I propose we remove both.

Assigning to Jeff for TL decision.

### PR DESCRIPTION
**Pull request by dannychapman**
Fixes #951

### CODE REVIEW

**Comment by dannychapman** [comment]
Please review  @awong-dev

File: _includes/header.html [hunk 1]
```diff
@@ -154,14 +154,9 @@ <h1 role="banner"><a href="/">Vets.gov</a></h1>
...
-      var usasearch_config = { siteHandle:"vets.gov_staging" };
-
-      var script = document.createElement("script");
-      script.type = "text/javascript";
-      script.src = "https://search.usa.gov/javascripts/remote.loader.js";
-      document.getElementsByTagName("head")[0].appendChild(script);
-
-//]]>
+<script>
+    var usasearch_config = { siteHandle:"vets.gov_staging" };
```
Comment (awong-dev) [inline _includes/header.html]: 2 space indent seems to be the prevailing standard. Reindent please?

**Comment by dannychapman** [comment]
@awong-dev Done!

**Comment by awong-dev** [comment]
LGTM!

**Pull request merged**